src/main/java/tempest/ui/pages/HomePage.java

```diff
@@ -13,8 +13,8 @@
 
     private final GUIManager manager;
 
-    private JButton manageModulesButton;
-    private JButton addSessionButton;
+    private final JButton manageModulesButton;
```

Comment:
The type of these can be changed to `LinkButton` as well right?

Reply:
I've merged this branch with sprint 2 so these are now LinkButtons